### DISCUSSION TITLE
Is there a mod that lets you disable the tray overflow?

### DISCUSSION THREAD

**Discussion by Xalaxis** [ramensoftware/windhawk-mods#793] [Q&A]
One of the things that has always peeved me about Win11 is the fact there is no way to just tell it to always display all tray icons, you have to manually approve each one - This is extra annoying with apps like Discord which keep changing their executable path so keep getting hidden after each update. I had a quick look through, but I don't think there are any mods which just do this at present?

**Comment by m417z** [comment]
Hi, there is now, check out [Always show all taskbar tray icons](https://windhawk.net/mods/taskbar-notification-icons-show-all).
